.travis.yml

```diff
@@ -25,4 +26,4 @@ install:
 
 script:
     - php bin/phpspec run
-    - php bin/phpunit
+    - php bin/simple-phpunit
```

Comment:
What is simple-phpunit? :)

Reply:
It is provided by symfony/phpunit-bridge. Allows you to run your test suite built with older versions of phpunit (in this case symfony 2.7)